dev-tools/iota-names-indexer/assets/grafana/datasources.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: 1
+
+datasources:
```

Comment:
This makes it so we automatically have a data source from prometheus in grafana available?

Reply:
Yeah basically